### PR TITLE
Feature/list all orders

### PR DESCRIPTION
**Pull request by caiogaspar**
Adding feature to get an order list using filters and pagination.

### CODE REVIEW

**Comment by SOSTheBlack** [comment]
Document the functions

File: src/Helper/Pagination.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Moip\Helper;
+
+class Pagination
```
Comment (joaolucasl) [inline src/Helper/Pagination.php]: This file needs to be documented, even though the function names are fairly self explanatory :)

File: src/Helper/Filters.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Moip\Helper;
+
+class Filters
```
Comment (joaolucasl) [inline src/Helper/Filters.php]: Same as `\Pagination` :)

**Comment by joaolucasl** [comment]
@SOSTheBlack - I don't think this PR should be closed? @caiogaspar is still working on the changes :)

**Comment by SOSTheBlack** [comment]
@joaolucasl Show \o/

**Comment by SOSTheBlack** [comment]
@caiogaspar Just need to document all the functions you have created ;)

**Pull request merged**